pkg/etl/doc.txt

```diff
@@ -0,0 +1,11 @@
+/**
+ * @defgroup cpp_etl Embedded Template Library
+ * @ingroup  cpp
```

Comment:
Since it's a package it should at least be listed under the pkg group. You can also put it under cpp. The group name should be pkg_etl IMHO.